### DISCUSSION TITLE
Problem with PlantUML web site: Can't get rid of Cookies nag screen.

### DISCUSSION THREAD

**Discussion by JohEble** [plantuml/plantuml#1350] [General]
Hello community,

there might be a problem with the PlantUML web site (https://plantuml.com). First you will be asked to accept that the site uses cookies, as usual. The problem is, that all the buttons in the browser such as "Weiter mit empfohlenen Cookies" (I don't know the english text, it translates to "Continue with recommended cookies") do not have any effect. So the Cookies nag screen keeps being toplevel, hiding or darkening the real content. I tried several browser, also, other persons have the same problem. Only if you browse the site in an old smart phone, the web site is usable. Is this a known problem? It definitely worked fine last year.

I would greately appreciate a solution to this problem since we also use PlantUML for education purposes.

Best regards

Johannes

**Comment by evantill** [comment]
Hi, Have you some ad blockers ?

If this is the case, try without it.

**Reply by JohEble** [reply]
Hi, I have checked Firefox, and there are no adblockers. And even if this was the reason, there should be a message showing that my browser blocks something, just like some newspapers or other sites have it.

**Reply by evantill** [reply]
We have nothing to block ad blocker, just that sometimes it's related

I can not reproduce it.

**Comment by arnaudroques** [comment]
Could try again? (maybe  clean the cache)
It should be better now

**Reply by JohEble** [reply]
Oh yes, it is good now. Thanks very much.

Best regards

Johannes

**Comment by evantill** [comment]
Maybe try using a new private window ?
